src/utils/constans.ts

```diff
@@ -0,0 +1,24 @@
+enum SearchCityTypes {
+    CITY = 'city',
+    AREA = 'area',
+}
+enum SearchCityCircle {
+    CIRCLE = '53.90,27.55',
+    RADIUS = 300000,
+}
+enum SearchCityCountryCode {
+    COUNTRY_CODE = 'BLR',
+}
+enum SearchCityLimit {
+    LIMIT = 10,
+}
+enum SearchCityLang {
+    LANG = 'ru',
+}
```

Comment:
wrong! Read docs about why do we need enums and constants, what are they used for. And read my [comment](https://github.com/brAnna2000/Current-Weather/pull/3/files#r792646630) more attentively.